### PR TITLE
Quickstart updates due to driver support changes

### PR DESCRIPTION
**Pull request by goldenx86**
Intel dropped support for pre-700 series GPUs, so Windows recommendations got updated.
Changed recommended dedicated GPUs after driver improvements and prices got better.
The previous minimum AMD GPU recommended for Linux was only included on CPUs that can't realistically run yuzu.
Expanded CPU support a bit to mention FX's abismal performance.
Added recommendations regarding VRAM, VP9 decoding.
Increased Vulkan requirement to 1.2.
And a bit more.

### CODE REVIEW

File: site/content/help/quickstart/index.md [hunk 1]
```diff
@@ -30,36 +30,36 @@ description: A guide designed to get you started with yuzu quickly.
 
 #### CPU: 
 
-Any x86_64 CPU with support for the FMA instruction set. 6 threads or more are recommended.
+Any x86_64 CPU with support for the FMA and AVX2 instruction set. 6 threads or more are recommended. Pre-Ryzen CPUs are strongly not recommended.
```
Comment (CaptV0rt3x) [inline site/content/help/quickstart/index.md:33]: `Pre-Ryzen AMD CPUs` ?

File: site/content/help/quickstart/index.md [hunk 2]
```diff
@@ -30,36 +30,36 @@ description: A guide designed to get you started with yuzu quickly.
...
 
-- Minimum for Windows: Intel HD Graphics 520 / AMD Radeon Vega 3
+- Minimum for Windows: Intel UHD Graphics 730 / AMD Radeon Vega 3
 
-- Recommended: Intel UHD Graphics 750 / AMD Radeon Vega 7
+- Recommended: Intel Xe Graphics / AMD Radeon Graphics (Vega 8)
 
 #### RAM: 
 
-Since an integrated GPU uses system RAM as its video memory (VRAM), our memory requirement in this configuration is higher.
```
Comment (CaptV0rt3x) [inline site/content/help/quickstart/index.md:62]: is this change really needed?

**Review by CaptV0rt3x — CHANGES_REQUESTED** [review]

File: site/content/help/quickstart/index.md (hunk 1 above)
Comment (goldenx86) [reply site/content/help/quickstart/index.md:33]: FX and older are slow AF.

File: site/content/help/quickstart/index.md (hunk 2 above)
Comment (goldenx86) [reply site/content/help/quickstart/index.md:62]: Reordered where what VRAM means so it got modified.

File: site/content/help/quickstart/index.md (hunk 1 above)
Comment (CaptV0rt3x) [reply site/content/help/quickstart/index.md:33]: I meant you should add "AMD"
